RadixWallet/Prelude/Extensions/Array+Identifiable.swift

```diff
@@ -1,9 +1,16 @@
 
 extension Array where Element: Identifiable {
 	/// Returns an `IdentifiedArray` of the `Element`, omitting clashing elements
-	public func asIdentifiable() -> IdentifiedArrayOf<Element> {
+	public func asIdentified() -> IdentifiedArrayOf<Element> {
 		var array: IdentifiedArrayOf<Element> = []
-		array.append(contentsOf: self)
+		for element in self {
+			let (inserted, _) = array.append(element)
+			if !inserted {
+				#if DEBUG
+				assertionFailure("The source array does not contain unique elements, id clash for \(element.id)")
+				#endif
```

Comment:
isn't the `#if DEBUG` redundant?

Also, we could merge the if condition with the assertion and just do:
```Swift
assert(inserted, "The source array does not contain unique elements, id clash for \(element.id)")
```

Reply:
correct, forgot to clean it up after I pivoted to this solution